homeassistant/components/binary_sensor/wink.py

```diff
@@ -0,0 +1,78 @@
+"""
+Support for Wink sensors.
+
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.wink/
+"""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import CONF_ACCESS_TOKEN, STATE_ON, STATE_OFF
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['python-wink==0.6.1']
+
+BINARY_TYPES = ['brightness',
```

Comment:
I would suggest you make this a dictionary mapping type to sensor class. This will get you the right icons in the ui